2018-11-21 20:20:26,521 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:26,606 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:26,627 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:27,310 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.1:kss-automap-lane:20529
2018-11-21 20:20:27,310 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.1:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:20:27,311 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.1:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:27,395 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:27,486 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:27,501 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:28,154 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.1:kss-automap-lane:20528
2018-11-21 20:20:28,155 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.1:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:20:28,155 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.1:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:28,247 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:28,300 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:28,343 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:29,059 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.81:kss-automap-lane:20527
2018-11-21 20:20:29,059 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.81:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:20:29,060 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.81:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:29,142 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:29,196 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:29,235 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:29,926 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.81:kss-automap-lane:20528
2018-11-21 20:20:29,927 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.81:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:20:29,927 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.81:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:30,013 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:30,099 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:30,121 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:30,833 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.1:kss-automap-lane:20527
2018-11-21 20:20:30,834 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.1:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:20:30,834 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.1:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:30,922 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:30,975 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:31,019 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:31,699 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.81:kss-automap-lane:20529
2018-11-21 20:20:31,699 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.81:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:20:31,699 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.81:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:31,792 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:31,843 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:31,887 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:32,592 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.22:kss-automap-lane:20527
2018-11-21 20:20:32,593 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.22:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:20:32,593 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.22:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:32,683 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:32,773 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:32,794 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:33,527 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.22:kss-automap-lane:20528
2018-11-21 20:20:33,527 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.22:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:20:33,528 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.22:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:33,614 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:33,702 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:33,726 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:34,392 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.22:kss-automap-lane:20529
2018-11-21 20:20:34,392 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.22:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:20:34,392 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.22:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:34,497 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:34,590 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:34,609 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:35,300 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.109:kss-automap-lane:20531
2018-11-21 20:20:35,300 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.109:kss-automap-lane:20531 failed, version mismatch
2018-11-21 20:20:35,300 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.109:kss-automap-lane:20531 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:35,395 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:35,445 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:35,487 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:36,180 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.109:kss-automap-lane:20530
2018-11-21 20:20:36,181 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.109:kss-automap-lane:20530 failed, version mismatch
2018-11-21 20:20:36,181 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.109:kss-automap-lane:20530 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:36,278 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:36,369 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:36,390 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:37,031 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.13:kss-automap-lane:20528
2018-11-21 20:20:37,031 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.13:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:20:37,032 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.13:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:37,130 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:37,221 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:37,239 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:37,898 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.13:kss-automap-lane:20527
2018-11-21 20:20:37,898 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.13:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:20:37,898 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.13:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:38,001 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:38,088 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:38,109 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:38,820 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.40:kss-automap-lane:20531
2018-11-21 20:20:38,820 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.40:kss-automap-lane:20531 failed, version mismatch
2018-11-21 20:20:38,820 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.40:kss-automap-lane:20531 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:38,916 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:39,005 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:39,026 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:39,668 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.13:kss-automap-lane:20529
2018-11-21 20:20:39,668 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.13:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:20:39,668 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.13:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:39,764 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:39,817 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:39,858 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:40,593 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.40:kss-automap-lane:20530
2018-11-21 20:20:40,593 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.40:kss-automap-lane:20530 failed, version mismatch
2018-11-21 20:20:40,593 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.40:kss-automap-lane:20530 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:40,685 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:40,769 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:40,793 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:41,351 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.38:kss-automap-lane:20527
2018-11-21 20:20:41,352 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.38:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:20:41,352 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.38:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:41,438 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:41,527 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:41,548 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:42,106 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.38:kss-automap-lane:20529
2018-11-21 20:20:42,106 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.38:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:20:42,107 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.38:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:42,194 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:42,281 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:42,302 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:42,844 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.38:kss-automap-lane:20528
2018-11-21 20:20:42,845 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.38:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:20:42,845 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.38:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:42,943 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:42,996 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:43,035 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:43,745 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.40:kss-automap-lane:20528
2018-11-21 20:20:43,745 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.40:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:20:43,745 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.40:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:43,841 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:43,896 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:43,938 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:44,678 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.40:kss-automap-lane:20527
2018-11-21 20:20:44,678 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.40:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:20:44,678 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.40:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:44,774 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:44,862 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:44,876 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:45,505 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.79:kss-automap-lane:20527
2018-11-21 20:20:45,505 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.79:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:20:45,505 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.79:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:45,600 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:45,686 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:45,704 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:46,370 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.79:kss-automap-lane:20528
2018-11-21 20:20:46,370 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.79:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:20:46,371 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.79:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:46,472 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:46,558 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:46,579 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:47,194 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.79:kss-automap-lane:20529
2018-11-21 20:20:47,195 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.79:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:20:47,195 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.79:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:47,297 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:47,381 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:47,402 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:47,932 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.83:kss-automap-lane:20531
2018-11-21 20:20:47,932 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.83:kss-automap-lane:20531 failed, version mismatch
2018-11-21 20:20:47,932 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.83:kss-automap-lane:20531 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:48,017 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:48,102 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:48,128 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:48,733 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.63:kss-automap-lane:20529
2018-11-21 20:20:48,733 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.63:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:20:48,733 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.63:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:48,830 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:48,918 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:48,941 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:49,496 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.63:kss-automap-lane:20528
2018-11-21 20:20:49,496 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.63:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:20:49,496 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.63:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:49,595 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:49,652 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:49,696 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:50,374 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.40:kss-automap-lane:20529
2018-11-21 20:20:50,374 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.40:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:20:50,374 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.40:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:50,472 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:50,562 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:50,582 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:51,152 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.63:kss-automap-lane:20527
2018-11-21 20:20:51,152 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.63:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:20:51,152 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.63:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:51,247 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:51,336 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:51,354 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:52,059 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.38:kss-automap-lane:20530
2018-11-21 20:20:52,059 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.38:kss-automap-lane:20530 failed, version mismatch
2018-11-21 20:20:52,059 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.38:kss-automap-lane:20530 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:52,153 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:52,237 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:52,259 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:52,809 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.83:kss-automap-lane:20530
2018-11-21 20:20:52,809 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.83:kss-automap-lane:20530 failed, version mismatch
2018-11-21 20:20:52,809 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.83:kss-automap-lane:20530 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:52,925 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:52,978 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:53,017 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:53,641 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.38:kss-automap-lane:20531
2018-11-21 20:20:53,641 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.38:kss-automap-lane:20531 failed, version mismatch
2018-11-21 20:20:53,642 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.38:kss-automap-lane:20531 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:53,736 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:53,821 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:53,847 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:54,537 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.109:kss-automap-lane:20528
2018-11-21 20:20:54,537 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.109:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:20:54,537 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.109:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:54,629 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:54,681 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:54,716 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:55,420 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.109:kss-automap-lane:20527
2018-11-21 20:20:55,421 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.109:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:20:55,421 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.109:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:55,514 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:55,565 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:55,600 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:56,331 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.109:kss-automap-lane:20529
2018-11-21 20:20:56,331 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.109:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:20:56,332 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.109:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:56,433 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:56,519 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:56,541 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:57,253 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.33:kss-automap-lane:20530
2018-11-21 20:20:57,253 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.33:kss-automap-lane:20530 failed, version mismatch
2018-11-21 20:20:57,253 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.33:kss-automap-lane:20530 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:57,352 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:57,440 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:57,464 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:58,064 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.54:kss-automap-lane:20531
2018-11-21 20:20:58,065 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.54:kss-automap-lane:20531 failed, version mismatch
2018-11-21 20:20:58,065 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.54:kss-automap-lane:20531 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:58,360 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:58,448 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:58,471 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:20:59,174 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.20:kss-automap-lane:20528
2018-11-21 20:20:59,174 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.20:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:20:59,174 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.20:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:20:59,273 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:20:59,329 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:20:59,368 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:00,102 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.33:kss-automap-lane:20531
2018-11-21 20:21:00,103 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.33:kss-automap-lane:20531 failed, version mismatch
2018-11-21 20:21:00,103 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.33:kss-automap-lane:20531 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:00,207 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:00,261 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:00,295 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:00,983 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.20:kss-automap-lane:20527
2018-11-21 20:21:00,983 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.20:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:21:00,984 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.20:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:01,076 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:01,162 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:01,187 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:01,851 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.15:kss-automap-lane:20527
2018-11-21 20:21:01,851 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.15:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:21:01,851 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.15:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:01,947 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:02,034 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:02,056 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:02,731 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.15:kss-automap-lane:20528
2018-11-21 20:21:02,732 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.15:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:21:02,732 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.15:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:02,817 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:02,903 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:02,927 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:03,638 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.20:kss-automap-lane:20529
2018-11-21 20:21:03,639 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.20:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:21:03,639 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.20:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:03,735 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:03,823 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:03,846 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:04,483 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.54:kss-automap-lane:20530
2018-11-21 20:21:04,484 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.54:kss-automap-lane:20530 failed, version mismatch
2018-11-21 20:21:04,484 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.54:kss-automap-lane:20530 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:04,581 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:04,670 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:04,694 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:05,470 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.42:kss-automap-lane:20530
2018-11-21 20:21:05,470 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.42:kss-automap-lane:20530 failed, version mismatch
2018-11-21 20:21:05,470 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.42:kss-automap-lane:20530 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:05,567 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:05,622 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:05,665 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:06,490 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.42:kss-automap-lane:20531
2018-11-21 20:21:06,491 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.42:kss-automap-lane:20531 failed, version mismatch
2018-11-21 20:21:06,491 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.42:kss-automap-lane:20531 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:06,591 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:06,643 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:06,678 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:07,388 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.15:kss-automap-lane:20529
2018-11-21 20:21:07,389 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.15:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:21:07,389 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.15:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:07,487 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:07,574 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:07,593 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:08,145 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.79:kss-automap-lane:20530
2018-11-21 20:21:08,145 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.79:kss-automap-lane:20530 failed, version mismatch
2018-11-21 20:21:08,145 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.79:kss-automap-lane:20530 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:08,239 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:08,327 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:08,342 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:08,989 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.79:kss-automap-lane:20531
2018-11-21 20:21:08,990 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.79:kss-automap-lane:20531 failed, version mismatch
2018-11-21 20:21:08,990 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.79:kss-automap-lane:20531 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:09,092 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:09,182 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:09,205 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:09,943 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.81:kss-automap-lane:20530
2018-11-21 20:21:09,943 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.81:kss-automap-lane:20530 failed, version mismatch
2018-11-21 20:21:09,944 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.81:kss-automap-lane:20530 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:10,049 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:10,102 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:10,146 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:10,842 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.81:kss-automap-lane:20531
2018-11-21 20:21:10,843 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.81:kss-automap-lane:20531 failed, version mismatch
2018-11-21 20:21:10,843 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.81:kss-automap-lane:20531 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:10,942 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:10,993 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:11,038 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:11,772 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.42:kss-automap-lane:20527
2018-11-21 20:21:11,772 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.42:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:21:11,773 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.42:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:11,868 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:11,921 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:11,939 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:12,612 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.42:kss-automap-lane:20528
2018-11-21 20:21:12,612 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.42:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:21:12,612 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.42:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:12,722 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:12,772 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:12,816 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:13,499 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.42:kss-automap-lane:20529
2018-11-21 20:21:13,499 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.42:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:21:13,500 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.42:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:13,599 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:13,688 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:13,710 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:14,343 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.54:kss-automap-lane:20528
2018-11-21 20:21:14,344 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.54:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:21:14,344 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.54:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:14,442 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:14,531 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:14,553 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:15,164 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.54:kss-automap-lane:20529
2018-11-21 20:21:15,164 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.54:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:21:15,164 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.54:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:15,267 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:15,355 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:15,377 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:16,006 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.54:kss-automap-lane:20527
2018-11-21 20:21:16,007 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.54:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:21:16,007 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.54:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:16,103 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:16,158 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:16,201 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:16,897 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.35:kss-automap-lane:20531
2018-11-21 20:21:16,897 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.35:kss-automap-lane:20531 failed, version mismatch
2018-11-21 20:21:16,897 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.35:kss-automap-lane:20531 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:16,993 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:17,049 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:17,079 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:17,778 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.35:kss-automap-lane:20530
2018-11-21 20:21:17,778 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.35:kss-automap-lane:20530 failed, version mismatch
2018-11-21 20:21:17,779 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.35:kss-automap-lane:20530 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:17,879 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:17,968 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:17,990 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:18,606 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.61:kss-automap-lane:20531
2018-11-21 20:21:18,606 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.61:kss-automap-lane:20531 failed, version mismatch
2018-11-21 20:21:18,606 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.61:kss-automap-lane:20531 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:18,706 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:18,762 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:18,796 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:19,536 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.85:kss-automap-lane:20528
2018-11-21 20:21:19,537 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.85:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:21:19,537 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.85:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:19,637 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:19,693 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:19,732 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:20,456 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.85:kss-automap-lane:20529
2018-11-21 20:21:20,456 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.85:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:21:20,457 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.85:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:20,564 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:20,654 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:20,678 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:21,434 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.85:kss-automap-lane:20527
2018-11-21 20:21:21,434 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.85:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:21:21,434 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.85:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:21,532 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:21,621 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:21,645 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:22,210 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.44:kss-automap-lane:20531
2018-11-21 20:21:22,211 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.44:kss-automap-lane:20531 failed, version mismatch
2018-11-21 20:21:22,211 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.44:kss-automap-lane:20531 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:22,310 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:22,359 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:22,405 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:23,206 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.44:kss-automap-lane:20530
2018-11-21 20:21:23,206 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.44:kss-automap-lane:20530 failed, version mismatch
2018-11-21 20:21:23,206 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.44:kss-automap-lane:20530 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:23,306 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:23,392 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:23,416 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:24,015 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.61:kss-automap-lane:20530
2018-11-21 20:21:24,016 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.61:kss-automap-lane:20530 failed, version mismatch
2018-11-21 20:21:24,016 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.61:kss-automap-lane:20530 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:24,113 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:24,170 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:24,208 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:24,962 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.76:kss-automap-lane:20529
2018-11-21 20:21:24,962 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.76:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:21:24,962 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.76:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:25,061 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:25,114 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:25,156 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:25,829 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.76:kss-automap-lane:20528
2018-11-21 20:21:25,829 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.76:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:21:25,829 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.76:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:25,931 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:25,985 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:26,028 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:26,797 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.76:kss-automap-lane:20527
2018-11-21 20:21:26,798 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.76:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:21:26,798 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.76:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:26,895 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:26,986 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:27,010 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:27,769 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.76:kss-automap-lane:20531
2018-11-21 20:21:27,770 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.76:kss-automap-lane:20531 failed, version mismatch
2018-11-21 20:21:27,770 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.76:kss-automap-lane:20531 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:27,874 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:27,962 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:27,988 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:28,692 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.76:kss-automap-lane:20530
2018-11-21 20:21:28,692 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.76:kss-automap-lane:20530 failed, version mismatch
2018-11-21 20:21:28,692 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.76:kss-automap-lane:20530 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:28,794 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:28,848 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:28,892 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:29,685 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.44:kss-automap-lane:20528
2018-11-21 20:21:29,685 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.44:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:21:29,685 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.44:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:29,791 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:29,844 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:29,887 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:30,673 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.44:kss-automap-lane:20527
2018-11-21 20:21:30,674 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.44:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:21:30,674 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.44:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:30,775 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:30,828 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:30,871 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:31,654 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.44:kss-automap-lane:20529
2018-11-21 20:21:31,655 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.44:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:21:31,655 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.44:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:31,752 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:31,804 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:31,848 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:32,579 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.33:kss-automap-lane:20527
2018-11-21 20:21:32,580 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.33:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:21:32,580 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.33:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:32,699 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:32,789 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:32,812 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:33,532 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.33:kss-automap-lane:20528
2018-11-21 20:21:33,532 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.33:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:21:33,533 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.33:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:33,832 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:33,885 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:33,921 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:34,634 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.18:kss-automap-lane:20527
2018-11-21 20:21:34,634 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.18:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:21:34,634 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.18:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:34,732 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:34,821 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:34,841 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:35,542 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.18:kss-automap-lane:20529
2018-11-21 20:21:35,542 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.18:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:21:35,542 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.18:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:35,634 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:35,721 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:35,744 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:36,422 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.18:kss-automap-lane:20528
2018-11-21 20:21:36,423 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.18:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:21:36,423 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.18:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:36,526 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:36,581 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:36,623 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:37,357 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.33:kss-automap-lane:20529
2018-11-21 20:21:37,357 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.33:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:21:37,357 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.33:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:37,457 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:37,541 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:37,563 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:38,111 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.83:kss-automap-lane:20529
2018-11-21 20:21:38,112 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.83:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:21:38,112 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.83:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:38,210 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:38,302 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:38,315 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:38,816 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.83:kss-automap-lane:20528
2018-11-21 20:21:38,816 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.83:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:21:38,816 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.83:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:38,924 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:38,975 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:38,993 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:40,384 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.3:kss-automap-lane:20528
2018-11-21 20:21:40,384 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.3:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:21:40,384 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.3:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:40,690 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:40,744 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:40,777 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:42,206 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.3:kss-automap-lane:20529
2018-11-21 20:21:42,207 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.3:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:21:42,207 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.3:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:42,311 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:42,398 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:42,416 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:42,950 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.83:kss-automap-lane:20527
2018-11-21 20:21:42,951 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.83:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:21:42,951 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.83:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:43,040 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:43,126 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:43,150 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:43,846 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.11:kss-automap-lane:20529
2018-11-21 20:21:43,846 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.11:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:21:43,847 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.11:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:43,944 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:43,993 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:44,029 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:44,687 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.11:kss-automap-lane:20527
2018-11-21 20:21:44,687 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.11:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:21:44,687 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.11:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:44,789 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:44,877 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:44,894 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:45,557 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.11:kss-automap-lane:20528
2018-11-21 20:21:45,557 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.11:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:21:45,558 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.11:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:45,658 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:45,744 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:45,766 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:46,354 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.63:kss-automap-lane:20531
2018-11-21 20:21:46,354 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.63:kss-automap-lane:20531 failed, version mismatch
2018-11-21 20:21:46,354 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.63:kss-automap-lane:20531 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:46,450 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:46,535 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:46,562 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:47,137 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.63:kss-automap-lane:20530
2018-11-21 20:21:47,137 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.63:kss-automap-lane:20530 failed, version mismatch
2018-11-21 20:21:47,137 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.63:kss-automap-lane:20530 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:47,242 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:47,296 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:47,332 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:48,342 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.3:kss-automap-lane:20527
2018-11-21 20:21:48,343 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.3:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:21:48,343 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.3:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:48,442 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:48,493 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:48,536 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:49,283 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.74:kss-automap-lane:20527
2018-11-21 20:21:49,284 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.74:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:21:49,284 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.74:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:49,382 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:49,432 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:49,473 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:50,170 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.74:kss-automap-lane:20528
2018-11-21 20:21:50,170 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.74:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:21:50,170 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.74:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:50,268 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:50,321 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:50,364 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:51,102 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.74:kss-automap-lane:20529
2018-11-21 20:21:51,102 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.74:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:21:51,102 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.74:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:51,201 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:51,288 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:51,312 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:51,890 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.61:kss-automap-lane:20527
2018-11-21 20:21:51,890 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.61:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:21:51,890 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.61:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:51,993 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:52,045 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:52,089 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:52,814 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.74:kss-automap-lane:20530
2018-11-21 20:21:52,815 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.74:kss-automap-lane:20530 failed, version mismatch
2018-11-21 20:21:52,815 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.74:kss-automap-lane:20530 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:52,910 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:52,995 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:53,016 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:53,700 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.74:kss-automap-lane:20531
2018-11-21 20:21:53,701 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.74:kss-automap-lane:20531 failed, version mismatch
2018-11-21 20:21:53,701 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.74:kss-automap-lane:20531 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:53,822 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:53,908 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:53,933 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:54,565 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.61:kss-automap-lane:20529
2018-11-21 20:21:54,566 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.61:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:21:54,566 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.61:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:54,664 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:54,751 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:54,772 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:55,383 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.61:kss-automap-lane:20528
2018-11-21 20:21:55,383 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.61:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:21:55,384 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.61:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:55,490 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:55,545 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:55,587 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:56,396 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.85:kss-automap-lane:20531
2018-11-21 20:21:56,396 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.85:kss-automap-lane:20531 failed, version mismatch
2018-11-21 20:21:56,396 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.85:kss-automap-lane:20531 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:56,473 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:56,563 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:56,586 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:57,327 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.85:kss-automap-lane:20530
2018-11-21 20:21:57,328 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.85:kss-automap-lane:20530 failed, version mismatch
2018-11-21 20:21:57,328 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.85:kss-automap-lane:20530 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:57,433 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:57,486 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:57,529 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:58,161 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.35:kss-automap-lane:20528
2018-11-21 20:21:58,162 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.35:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:21:58,162 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.35:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:58,267 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:58,323 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:58,356 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:59,046 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.35:kss-automap-lane:20527
2018-11-21 20:21:59,046 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.35:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:21:59,047 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.35:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:21:59,148 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:21:59,236 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:21:59,256 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:21:59,975 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.35:kss-automap-lane:20529
2018-11-21 20:21:59,975 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.35:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:21:59,975 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.35:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:00,077 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:00,130 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:00,171 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:00,903 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.73:kss-automap-lane:20531
2018-11-21 20:22:00,904 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.73:kss-automap-lane:20531 failed, version mismatch
2018-11-21 20:22:00,904 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.73:kss-automap-lane:20531 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:01,000 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:01,090 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:01,113 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:01,823 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.73:kss-automap-lane:20530
2018-11-21 20:22:01,824 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.73:kss-automap-lane:20530 failed, version mismatch
2018-11-21 20:22:01,824 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.73:kss-automap-lane:20530 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:01,926 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:02,013 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:02,038 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:02,633 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.64:kss-automap-lane:20527
2018-11-21 20:22:02,634 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.64:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:22:02,634 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.64:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:02,735 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:02,820 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:02,846 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:03,448 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.64:kss-automap-lane:20528
2018-11-21 20:22:03,448 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.64:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:22:03,448 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.64:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:03,551 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:03,639 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:03,661 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:04,411 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.82:kss-automap-lane:20530
2018-11-21 20:22:04,411 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.82:kss-automap-lane:20530 failed, version mismatch
2018-11-21 20:22:04,412 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.82:kss-automap-lane:20530 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:04,512 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:04,597 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:04,623 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:05,243 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.64:kss-automap-lane:20529
2018-11-21 20:22:05,243 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.64:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:22:05,244 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.64:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:05,347 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:05,434 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:05,458 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:06,134 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.82:kss-automap-lane:20531
2018-11-21 20:22:06,135 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.82:kss-automap-lane:20531 failed, version mismatch
2018-11-21 20:22:06,135 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.82:kss-automap-lane:20531 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:06,235 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:06,289 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:06,324 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:07,055 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.39:kss-automap-lane:20531
2018-11-21 20:22:07,055 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.39:kss-automap-lane:20531 failed, version mismatch
2018-11-21 20:22:07,055 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.39:kss-automap-lane:20531 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:07,154 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:07,245 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:07,267 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:08,014 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.39:kss-automap-lane:20530
2018-11-21 20:22:08,014 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.39:kss-automap-lane:20530 failed, version mismatch
2018-11-21 20:22:08,014 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.39:kss-automap-lane:20530 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:08,114 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:08,203 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:08,223 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:08,908 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.82:kss-automap-lane:20529
2018-11-21 20:22:08,908 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.82:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:22:08,908 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.82:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:09,012 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:09,097 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:09,122 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:09,806 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.21:kss-automap-lane:20528
2018-11-21 20:22:09,806 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.21:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:22:09,807 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.21:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:09,902 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:09,990 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:10,013 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:10,659 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.82:kss-automap-lane:20527
2018-11-21 20:22:10,659 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.82:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:22:10,660 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.82:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:10,762 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:10,850 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:10,870 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:11,554 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.82:kss-automap-lane:20528
2018-11-21 20:22:11,554 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.82:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:22:11,554 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.82:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:11,657 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:11,708 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:11,746 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:12,407 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.21:kss-automap-lane:20529
2018-11-21 20:22:12,408 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.21:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:22:12,408 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.21:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:12,509 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:12,596 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:12,620 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:13,249 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.2:kss-automap-lane:20527
2018-11-21 20:22:13,250 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.2:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:22:13,250 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.2:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:13,356 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:13,445 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:13,468 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:14,188 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.14:kss-automap-lane:20527
2018-11-21 20:22:14,189 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.14:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:22:14,189 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.14:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:14,283 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:14,367 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:14,390 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:15,090 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.2:kss-automap-lane:20528
2018-11-21 20:22:15,090 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.2:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:22:15,090 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.2:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:15,182 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:15,270 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:15,292 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:15,893 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.21:kss-automap-lane:20527
2018-11-21 20:22:15,894 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.21:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:22:15,894 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.21:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:15,997 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:16,084 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:16,108 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:16,899 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.2:kss-automap-lane:20529
2018-11-21 20:22:16,899 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.2:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:22:16,899 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.2:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:16,998 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:17,085 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:17,110 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:17,684 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.64:kss-automap-lane:20530
2018-11-21 20:22:17,684 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.64:kss-automap-lane:20530 failed, version mismatch
2018-11-21 20:22:17,684 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.64:kss-automap-lane:20530 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:17,791 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:17,880 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:17,898 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:18,537 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.71:kss-automap-lane:20527
2018-11-21 20:22:18,537 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.71:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:22:18,538 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.71:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:18,642 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:18,729 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:18,755 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:19,357 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.64:kss-automap-lane:20531
2018-11-21 20:22:19,358 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.64:kss-automap-lane:20531 failed, version mismatch
2018-11-21 20:22:19,358 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.64:kss-automap-lane:20531 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:19,459 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:19,547 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:19,573 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:20,299 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.71:kss-automap-lane:20528
2018-11-21 20:22:20,299 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.71:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:22:20,300 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.71:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:20,399 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:20,451 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:20,490 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:21,227 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.71:kss-automap-lane:20529
2018-11-21 20:22:21,228 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.71:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:22:21,228 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.71:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:21,330 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:21,385 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:21,426 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:22,218 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.41:kss-automap-lane:20531
2018-11-21 20:22:22,218 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.41:kss-automap-lane:20531 failed, version mismatch
2018-11-21 20:22:22,218 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.41:kss-automap-lane:20531 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:22,324 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:22,379 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:22,409 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:23,109 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.14:kss-automap-lane:20529
2018-11-21 20:22:23,109 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.14:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:22:23,109 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.14:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:23,208 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:23,260 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:23,307 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:24,060 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.41:kss-automap-lane:20530
2018-11-21 20:22:24,060 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.41:kss-automap-lane:20530 failed, version mismatch
2018-11-21 20:22:24,061 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.41:kss-automap-lane:20530 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:24,167 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:24,219 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:24,259 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:24,964 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.14:kss-automap-lane:20528
2018-11-21 20:22:24,965 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.14:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:22:24,965 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.14:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:25,069 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:25,120 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:25,166 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:25,962 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.41:kss-automap-lane:20529
2018-11-21 20:22:25,963 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.41:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:22:25,963 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.41:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:26,059 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:26,147 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:26,172 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:26,924 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.78:kss-automap-lane:20529
2018-11-21 20:22:26,924 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.78:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:22:26,924 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.78:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:27,033 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:27,088 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:27,129 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:27,900 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.41:kss-automap-lane:20528
2018-11-21 20:22:27,901 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.41:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:22:27,901 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.41:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:28,008 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:28,061 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:28,104 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:28,824 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.78:kss-automap-lane:20528
2018-11-21 20:22:28,824 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.78:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:22:28,824 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.78:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:28,920 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:28,972 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:29,017 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:29,788 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.41:kss-automap-lane:20527
2018-11-21 20:22:29,788 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.41:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:22:29,789 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.41:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:29,891 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:29,944 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:29,988 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:30,741 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.78:kss-automap-lane:20527
2018-11-21 20:22:30,741 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.78:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:22:30,742 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.78:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:30,853 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:30,940 INFO 149329[transport.py:1746 _log]: Authentication (publickey) successful!
2018-11-21 20:22:31,606 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.80:kss-automap-lane:20530
2018-11-21 20:22:31,607 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.80:kss-automap-lane:20530 failed, version mismatch
2018-11-21 20:22:31,607 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.80:kss-automap-lane:20530 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:31,707 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:31,793 INFO 149329[transport.py:1746 _log]: Authentication (publickey) successful!
2018-11-21 20:22:32,462 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.80:kss-automap-lane:20531
2018-11-21 20:22:32,463 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.80:kss-automap-lane:20531 failed, version mismatch
2018-11-21 20:22:32,463 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.80:kss-automap-lane:20531 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:32,563 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:32,652 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:32,672 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:33,424 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.71:kss-automap-lane:20530
2018-11-21 20:22:33,424 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.71:kss-automap-lane:20530 failed, version mismatch
2018-11-21 20:22:33,424 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.71:kss-automap-lane:20530 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:33,546 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:33,633 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:33,661 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:34,405 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.71:kss-automap-lane:20531
2018-11-21 20:22:34,406 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.71:kss-automap-lane:20531 failed, version mismatch
2018-11-21 20:22:34,406 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.71:kss-automap-lane:20531 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:34,508 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:34,560 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:34,605 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:35,372 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.32:kss-automap-lane:20527
2018-11-21 20:22:35,372 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.32:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:22:35,372 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.32:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:35,470 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:35,523 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:35,564 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:36,332 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.32:kss-automap-lane:20528
2018-11-21 20:22:36,333 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.32:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:22:36,333 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.32:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:36,434 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:36,487 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:36,534 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:37,263 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.32:kss-automap-lane:20529
2018-11-21 20:22:37,264 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.32:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:22:37,264 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.32:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:37,366 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:37,452 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:37,478 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:38,148 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.16:kss-automap-lane:20528
2018-11-21 20:22:38,149 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.16:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:22:38,149 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.16:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:38,250 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:38,301 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:38,336 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:39,036 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.16:kss-automap-lane:20529
2018-11-21 20:22:39,036 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.16:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:22:39,036 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.16:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:39,131 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:39,224 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:39,247 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:40,008 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.32:kss-automap-lane:20530
2018-11-21 20:22:40,008 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.32:kss-automap-lane:20530 failed, version mismatch
2018-11-21 20:22:40,008 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.32:kss-automap-lane:20530 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:40,106 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:40,157 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:40,193 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:40,859 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.16:kss-automap-lane:20527
2018-11-21 20:22:40,859 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.16:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:22:40,860 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.16:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:40,967 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:41,020 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:41,063 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:41,829 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.32:kss-automap-lane:20531
2018-11-21 20:22:41,829 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.32:kss-automap-lane:20531 failed, version mismatch
2018-11-21 20:22:41,829 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.32:kss-automap-lane:20531 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:41,926 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:42,013 INFO 149329[transport.py:1746 _log]: Authentication (publickey) successful!
2018-11-21 20:22:42,724 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.80:kss-automap-lane:20529
2018-11-21 20:22:42,725 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.80:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:22:42,725 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.80:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:42,826 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:42,921 INFO 149329[transport.py:1746 _log]: Authentication (publickey) successful!
2018-11-21 20:22:43,536 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.80:kss-automap-lane:20527
2018-11-21 20:22:43,537 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.80:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:22:43,537 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.80:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:43,639 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:43,726 INFO 149329[transport.py:1746 _log]: Authentication (publickey) successful!
2018-11-21 20:22:44,368 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.80:kss-automap-lane:20528
2018-11-21 20:22:44,368 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.80:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:22:44,368 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.80:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:44,476 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:44,528 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:44,571 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:45,315 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.43:kss-automap-lane:20530
2018-11-21 20:22:45,315 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.43:kss-automap-lane:20530 failed, version mismatch
2018-11-21 20:22:45,316 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.43:kss-automap-lane:20530 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:45,412 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:45,501 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:45,527 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:46,222 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.43:kss-automap-lane:20531
2018-11-21 20:22:46,222 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.43:kss-automap-lane:20531 failed, version mismatch
2018-11-21 20:22:46,223 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.43:kss-automap-lane:20531 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:46,325 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:46,377 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:46,419 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:47,081 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.78:kss-automap-lane:20531
2018-11-21 20:22:47,082 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.78:kss-automap-lane:20531 failed, version mismatch
2018-11-21 20:22:47,082 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.78:kss-automap-lane:20531 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:47,184 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:47,236 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:47,278 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:48,025 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.78:kss-automap-lane:20530
2018-11-21 20:22:48,025 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.78:kss-automap-lane:20530 failed, version mismatch
2018-11-21 20:22:48,026 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.78:kss-automap-lane:20530 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:48,127 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:48,182 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:48,226 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:48,964 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.39:kss-automap-lane:20529
2018-11-21 20:22:48,965 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.39:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:22:48,965 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.39:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:49,073 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:49,124 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:49,161 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:49,866 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.39:kss-automap-lane:20528
2018-11-21 20:22:49,867 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.39:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:22:49,867 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.39:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:49,972 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:50,025 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:50,060 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:50,809 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.39:kss-automap-lane:20527
2018-11-21 20:22:50,809 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.39:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:22:50,810 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.39:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:50,912 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:51,002 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:51,022 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:51,737 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.17:kss-automap-lane:20529
2018-11-21 20:22:51,737 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.17:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:22:51,737 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.17:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:51,840 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:51,929 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:51,953 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:52,653 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.17:kss-automap-lane:20528
2018-11-21 20:22:52,653 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.17:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:22:52,654 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.17:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:52,753 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:52,840 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:52,863 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:53,536 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.17:kss-automap-lane:20527
2018-11-21 20:22:53,536 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.17:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:22:53,536 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.17:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:53,664 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:53,753 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:53,779 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:54,492 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.43:kss-automap-lane:20527
2018-11-21 20:22:54,493 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.43:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:22:54,493 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.43:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:54,599 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:54,689 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:54,713 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:55,486 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.43:kss-automap-lane:20528
2018-11-21 20:22:55,487 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.43:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:22:55,487 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.43:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:55,589 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:55,641 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:55,683 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:56,414 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.43:kss-automap-lane:20529
2018-11-21 20:22:56,414 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.43:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:22:56,414 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.43:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:56,699 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:56,780 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:56,799 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:57,387 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.51:kss-automap-lane:20528
2018-11-21 20:22:57,387 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.51:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:22:57,387 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.51:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:57,486 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:57,571 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:57,597 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:58,139 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.51:kss-automap-lane:20527
2018-11-21 20:22:58,140 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.51:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:22:58,140 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.51:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:58,250 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:58,341 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:58,363 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:59,006 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.34:kss-automap-lane:20529
2018-11-21 20:22:59,006 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.34:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:22:59,006 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.34:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:59,112 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:22:59,162 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:22:59,201 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:22:59,893 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.34:kss-automap-lane:20528
2018-11-21 20:22:59,893 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.34:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:22:59,893 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.34:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:22:59,979 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:00,029 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:00,064 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:00,719 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.34:kss-automap-lane:20527
2018-11-21 20:23:00,719 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.34:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:23:00,719 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.34:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:00,823 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:00,908 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:00,935 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:01,530 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.51:kss-automap-lane:20529
2018-11-21 20:23:01,530 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.51:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:23:01,530 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.51:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:01,625 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:01,677 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:01,717 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:02,457 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.34:kss-automap-lane:20531
2018-11-21 20:23:02,457 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.34:kss-automap-lane:20531 failed, version mismatch
2018-11-21 20:23:02,457 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.34:kss-automap-lane:20531 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:02,553 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:02,643 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:02,661 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:03,339 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.34:kss-automap-lane:20530
2018-11-21 20:23:03,339 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.34:kss-automap-lane:20530 failed, version mismatch
2018-11-21 20:23:03,339 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.34:kss-automap-lane:20530 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:03,438 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:03,492 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:03,535 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:04,245 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.75:kss-automap-lane:20529
2018-11-21 20:23:04,245 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.75:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:23:04,245 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.75:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:04,345 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:04,433 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:04,459 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:05,221 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.75:kss-automap-lane:20527
2018-11-21 20:23:05,221 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.75:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:23:05,221 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.75:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:05,329 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:05,417 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:05,443 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:06,188 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.75:kss-automap-lane:20528
2018-11-21 20:23:06,188 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.75:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:23:06,188 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.75:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:06,294 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:06,381 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:06,406 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:06,993 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.62:kss-automap-lane:20529
2018-11-21 20:23:06,993 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.62:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:23:06,993 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.62:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:07,097 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:07,185 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:07,200 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:07,868 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.36:kss-automap-lane:20527
2018-11-21 20:23:07,868 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.36:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:23:07,869 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.36:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:07,974 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:08,029 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:08,070 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:08,801 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.75:kss-automap-lane:20530
2018-11-21 20:23:08,801 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.75:kss-automap-lane:20530 failed, version mismatch
2018-11-21 20:23:08,801 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.75:kss-automap-lane:20530 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:08,905 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:08,992 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:09,017 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:09,778 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.75:kss-automap-lane:20531
2018-11-21 20:23:09,778 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.75:kss-automap-lane:20531 failed, version mismatch
2018-11-21 20:23:09,778 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.75:kss-automap-lane:20531 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:09,885 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:09,972 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:09,997 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:10,542 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.62:kss-automap-lane:20528
2018-11-21 20:23:10,542 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.62:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:23:10,542 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.62:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:10,645 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:10,733 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:10,755 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:11,267 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.62:kss-automap-lane:20527
2018-11-21 20:23:11,267 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.62:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:23:11,267 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.62:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:11,373 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:11,457 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:11,478 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:11,932 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.84:kss-automap-lane:20531
2018-11-21 20:23:11,932 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.84:kss-automap-lane:20531 failed, version mismatch
2018-11-21 20:23:11,932 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.84:kss-automap-lane:20531 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:12,029 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:12,112 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:12,134 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:12,601 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.84:kss-automap-lane:20530
2018-11-21 20:23:12,601 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.84:kss-automap-lane:20530 failed, version mismatch
2018-11-21 20:23:12,602 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.84:kss-automap-lane:20530 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:12,730 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:12,818 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:12,843 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:13,564 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.86:kss-automap-lane:20527
2018-11-21 20:23:13,564 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.86:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:23:13,564 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.86:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:13,666 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:13,754 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:13,781 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:14,513 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.86:kss-automap-lane:20528
2018-11-21 20:23:14,513 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.86:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:23:14,514 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.86:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:14,618 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:14,706 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:14,733 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:15,487 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.86:kss-automap-lane:20529
2018-11-21 20:23:15,487 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.86:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:23:15,487 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.86:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:15,592 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:15,680 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:15,702 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:16,396 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.19:kss-automap-lane:20527
2018-11-21 20:23:16,396 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.19:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:23:16,396 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.19:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:16,495 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:16,585 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:16,607 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:17,312 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.19:kss-automap-lane:20528
2018-11-21 20:23:17,312 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.19:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:23:17,312 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.19:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:17,421 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:17,508 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:17,528 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:18,182 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.36:kss-automap-lane:20528
2018-11-21 20:23:18,182 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.36:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:23:18,182 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.36:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:18,291 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:18,341 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:18,378 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:19,076 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.19:kss-automap-lane:20529
2018-11-21 20:23:19,076 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.19:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:23:19,076 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.19:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:19,182 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:19,273 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:19,290 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:19,873 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.36:kss-automap-lane:20529
2018-11-21 20:23:19,873 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.36:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:23:19,874 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.36:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:19,974 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:20,062 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:20,080 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:20,563 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.84:kss-automap-lane:20529
2018-11-21 20:23:20,563 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.84:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:23:20,564 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.84:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:20,662 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:20,713 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:20,744 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:21,303 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.36:kss-automap-lane:20531
2018-11-21 20:23:21,304 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.36:kss-automap-lane:20531 failed, version mismatch
2018-11-21 20:23:21,304 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.36:kss-automap-lane:20531 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:21,409 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:21,497 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:21,517 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:22,240 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.4:kss-automap-lane:20529
2018-11-21 20:23:22,241 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.4:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:23:22,241 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.4:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:22,347 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:22,436 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:22,458 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:23,032 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.51:kss-automap-lane:20531
2018-11-21 20:23:23,032 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.51:kss-automap-lane:20531 failed, version mismatch
2018-11-21 20:23:23,032 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.51:kss-automap-lane:20531 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:23,124 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:23,211 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:23,230 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:23,771 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.36:kss-automap-lane:20530
2018-11-21 20:23:23,772 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.36:kss-automap-lane:20530 failed, version mismatch
2018-11-21 20:23:23,772 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.36:kss-automap-lane:20530 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:24,075 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:24,160 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:24,185 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:24,783 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.51:kss-automap-lane:20530
2018-11-21 20:23:24,783 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.51:kss-automap-lane:20530 failed, version mismatch
2018-11-21 20:23:24,783 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.51:kss-automap-lane:20530 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:24,883 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:24,970 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:24,988 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:25,467 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.84:kss-automap-lane:20528
2018-11-21 20:23:25,467 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.84:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:23:25,467 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.84:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:25,568 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:25,652 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:25,674 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:26,143 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.84:kss-automap-lane:20527
2018-11-21 20:23:26,143 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.84:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:23:26,143 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.84:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:26,241 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:26,294 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:26,337 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:27,080 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.86:kss-automap-lane:20530
2018-11-21 20:23:27,080 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.86:kss-automap-lane:20530 failed, version mismatch
2018-11-21 20:23:27,080 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.86:kss-automap-lane:20530 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:27,184 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:27,234 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:27,280 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:27,972 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.86:kss-automap-lane:20531
2018-11-21 20:23:27,972 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.86:kss-automap-lane:20531 failed, version mismatch
2018-11-21 20:23:27,972 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.86:kss-automap-lane:20531 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:28,071 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:28,163 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:28,185 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:28,929 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.73:kss-automap-lane:20529
2018-11-21 20:23:28,930 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.73:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:23:28,930 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.73:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:29,037 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:29,123 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:29,147 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:29,826 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.12:kss-automap-lane:20529
2018-11-21 20:23:29,826 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.12:kss-automap-lane:20529 failed, version mismatch
2018-11-21 20:23:29,827 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.12:kss-automap-lane:20529 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:29,936 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:30,024 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:30,046 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:30,683 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.12:kss-automap-lane:20528
2018-11-21 20:23:30,683 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.12:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:23:30,683 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.12:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:30,793 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:30,881 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:30,905 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:31,502 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.62:kss-automap-lane:20531
2018-11-21 20:23:31,502 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.62:kss-automap-lane:20531 failed, version mismatch
2018-11-21 20:23:31,503 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.62:kss-automap-lane:20531 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:31,626 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:31,714 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:31,735 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:32,409 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.12:kss-automap-lane:20527
2018-11-21 20:23:32,410 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.12:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:23:32,410 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.12:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:32,513 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:32,563 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:32,601 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:33,181 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.62:kss-automap-lane:20530
2018-11-21 20:23:33,182 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.62:kss-automap-lane:20530 failed, version mismatch
2018-11-21 20:23:33,182 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.62:kss-automap-lane:20530 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:33,278 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:33,332 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:33,372 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:34,033 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.4:kss-automap-lane:20528
2018-11-21 20:23:34,034 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.4:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:23:34,034 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.4:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:34,137 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:34,229 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:34,249 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:34,944 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.4:kss-automap-lane:20527
2018-11-21 20:23:34,944 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.4:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:23:34,944 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.4:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:35,046 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:35,098 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:35,139 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:35,907 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.73:kss-automap-lane:20528
2018-11-21 20:23:35,908 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.73:kss-automap-lane:20528 failed, version mismatch
2018-11-21 20:23:35,908 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.73:kss-automap-lane:20528 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:36,008 INFO 149329[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-21 20:23:36,062 INFO 149329[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-21 20:23:36,104 INFO 149329[transport.py:1746 _log]: Authentication (password) successful!
2018-11-21 20:23:36,846 INFO 149329[check_automap_ver_v2.py:68 check_ver]: checking 10.11.5.73:kss-automap-lane:20527
2018-11-21 20:23:36,846 INFO 149329[check_automap_ver_v2.py:70 check_ver]: check 10.11.5.73:kss-automap-lane:20527 failed, version mismatch
2018-11-21 20:23:36,846 INFO 149329[check_automap_ver_v2.py:74 check_ver]: check 10.11.5.73:kss-automap-lane:20527 md5sum [u'1b96e911b235831b6fc19882afa87a0e  /data/CI/autohdmap_lane/autohdmap_lane_server\n']
2018-11-21 20:23:36,846 INFO 149329[check_automap_ver_v2.py:75 check_ver]: checked instance num 213
2018-11-21 20:23:36,847 INFO 149329[check_automap_ver_v2.py:83 main]: mismatch list [u'10.11.5.1:kss-automap-lane:20529', u'10.11.5.1:kss-automap-lane:20528', u'10.11.5.81:kss-automap-lane:20527', u'10.11.5.81:kss-automap-lane:20528', u'10.11.5.1:kss-automap-lane:20527', u'10.11.5.81:kss-automap-lane:20529', u'10.11.5.22:kss-automap-lane:20527', u'10.11.5.22:kss-automap-lane:20528', u'10.11.5.22:kss-automap-lane:20529', u'10.11.5.109:kss-automap-lane:20531', u'10.11.5.109:kss-automap-lane:20530', u'10.11.5.13:kss-automap-lane:20528', u'10.11.5.13:kss-automap-lane:20527', u'10.11.5.40:kss-automap-lane:20531', u'10.11.5.13:kss-automap-lane:20529', u'10.11.5.40:kss-automap-lane:20530', u'10.11.5.38:kss-automap-lane:20527', u'10.11.5.38:kss-automap-lane:20529', u'10.11.5.38:kss-automap-lane:20528', u'10.11.5.40:kss-automap-lane:20528', u'10.11.5.40:kss-automap-lane:20527', u'10.11.5.79:kss-automap-lane:20527', u'10.11.5.79:kss-automap-lane:20528', u'10.11.5.79:kss-automap-lane:20529', u'10.11.5.83:kss-automap-lane:20531', u'10.11.5.63:kss-automap-lane:20529', u'10.11.5.63:kss-automap-lane:20528', u'10.11.5.40:kss-automap-lane:20529', u'10.11.5.63:kss-automap-lane:20527', u'10.11.5.38:kss-automap-lane:20530', u'10.11.5.83:kss-automap-lane:20530', u'10.11.5.38:kss-automap-lane:20531', u'10.11.5.109:kss-automap-lane:20528', u'10.11.5.109:kss-automap-lane:20527', u'10.11.5.109:kss-automap-lane:20529', u'10.11.5.33:kss-automap-lane:20530', u'10.11.5.54:kss-automap-lane:20531', u'10.11.5.20:kss-automap-lane:20528', u'10.11.5.33:kss-automap-lane:20531', u'10.11.5.20:kss-automap-lane:20527', u'10.11.5.15:kss-automap-lane:20527', u'10.11.5.15:kss-automap-lane:20528', u'10.11.5.20:kss-automap-lane:20529', u'10.11.5.54:kss-automap-lane:20530', u'10.11.5.42:kss-automap-lane:20530', u'10.11.5.42:kss-automap-lane:20531', u'10.11.5.15:kss-automap-lane:20529', u'10.11.5.79:kss-automap-lane:20530', u'10.11.5.79:kss-automap-lane:20531', u'10.11.5.81:kss-automap-lane:20530', u'10.11.5.81:kss-automap-lane:20531', u'10.11.5.42:kss-automap-lane:20527', u'10.11.5.42:kss-automap-lane:20528', u'10.11.5.42:kss-automap-lane:20529', u'10.11.5.54:kss-automap-lane:20528', u'10.11.5.54:kss-automap-lane:20529', u'10.11.5.54:kss-automap-lane:20527', u'10.11.5.35:kss-automap-lane:20531', u'10.11.5.35:kss-automap-lane:20530', u'10.11.5.61:kss-automap-lane:20531', u'10.11.5.85:kss-automap-lane:20528', u'10.11.5.85:kss-automap-lane:20529', u'10.11.5.85:kss-automap-lane:20527', u'10.11.5.44:kss-automap-lane:20531', u'10.11.5.44:kss-automap-lane:20530', u'10.11.5.61:kss-automap-lane:20530', u'10.11.5.76:kss-automap-lane:20529', u'10.11.5.76:kss-automap-lane:20528', u'10.11.5.76:kss-automap-lane:20527', u'10.11.5.76:kss-automap-lane:20531', u'10.11.5.76:kss-automap-lane:20530', u'10.11.5.44:kss-automap-lane:20528', u'10.11.5.44:kss-automap-lane:20527', u'10.11.5.44:kss-automap-lane:20529', u'10.11.5.33:kss-automap-lane:20527', u'10.11.5.33:kss-automap-lane:20528', u'10.11.5.18:kss-automap-lane:20527', u'10.11.5.18:kss-automap-lane:20529', u'10.11.5.18:kss-automap-lane:20528', u'10.11.5.33:kss-automap-lane:20529', u'10.11.5.83:kss-automap-lane:20529', u'10.11.5.83:kss-automap-lane:20528', u'10.11.5.3:kss-automap-lane:20528', u'10.11.5.3:kss-automap-lane:20529', u'10.11.5.83:kss-automap-lane:20527', u'10.11.5.11:kss-automap-lane:20529', u'10.11.5.11:kss-automap-lane:20527', u'10.11.5.11:kss-automap-lane:20528', u'10.11.5.63:kss-automap-lane:20531', u'10.11.5.63:kss-automap-lane:20530', u'10.11.5.3:kss-automap-lane:20527', u'10.11.5.74:kss-automap-lane:20527', u'10.11.5.74:kss-automap-lane:20528', u'10.11.5.74:kss-automap-lane:20529', u'10.11.5.61:kss-automap-lane:20527', u'10.11.5.74:kss-automap-lane:20530', u'10.11.5.74:kss-automap-lane:20531', u'10.11.5.61:kss-automap-lane:20529', u'10.11.5.61:kss-automap-lane:20528', u'10.11.5.85:kss-automap-lane:20531', u'10.11.5.85:kss-automap-lane:20530', u'10.11.5.35:kss-automap-lane:20528', u'10.11.5.35:kss-automap-lane:20527', u'10.11.5.35:kss-automap-lane:20529', u'10.11.5.73:kss-automap-lane:20531', u'10.11.5.73:kss-automap-lane:20530', u'10.11.5.64:kss-automap-lane:20527', u'10.11.5.64:kss-automap-lane:20528', u'10.11.5.82:kss-automap-lane:20530', u'10.11.5.64:kss-automap-lane:20529', u'10.11.5.82:kss-automap-lane:20531', u'10.11.5.39:kss-automap-lane:20531', u'10.11.5.39:kss-automap-lane:20530', u'10.11.5.82:kss-automap-lane:20529', u'10.11.5.21:kss-automap-lane:20528', u'10.11.5.82:kss-automap-lane:20527', u'10.11.5.82:kss-automap-lane:20528', u'10.11.5.21:kss-automap-lane:20529', u'10.11.5.2:kss-automap-lane:20527', u'10.11.5.14:kss-automap-lane:20527', u'10.11.5.2:kss-automap-lane:20528', u'10.11.5.21:kss-automap-lane:20527', u'10.11.5.2:kss-automap-lane:20529', u'10.11.5.64:kss-automap-lane:20530', u'10.11.5.71:kss-automap-lane:20527', u'10.11.5.64:kss-automap-lane:20531', u'10.11.5.71:kss-automap-lane:20528', u'10.11.5.71:kss-automap-lane:20529', u'10.11.5.41:kss-automap-lane:20531', u'10.11.5.14:kss-automap-lane:20529', u'10.11.5.41:kss-automap-lane:20530', u'10.11.5.14:kss-automap-lane:20528', u'10.11.5.41:kss-automap-lane:20529', u'10.11.5.78:kss-automap-lane:20529', u'10.11.5.41:kss-automap-lane:20528', u'10.11.5.78:kss-automap-lane:20528', u'10.11.5.41:kss-automap-lane:20527', u'10.11.5.78:kss-automap-lane:20527', u'10.11.5.80:kss-automap-lane:20530', u'10.11.5.80:kss-automap-lane:20531', u'10.11.5.71:kss-automap-lane:20530', u'10.11.5.71:kss-automap-lane:20531', u'10.11.5.32:kss-automap-lane:20527', u'10.11.5.32:kss-automap-lane:20528', u'10.11.5.32:kss-automap-lane:20529', u'10.11.5.16:kss-automap-lane:20528', u'10.11.5.16:kss-automap-lane:20529', u'10.11.5.32:kss-automap-lane:20530', u'10.11.5.16:kss-automap-lane:20527', u'10.11.5.32:kss-automap-lane:20531', u'10.11.5.80:kss-automap-lane:20529', u'10.11.5.80:kss-automap-lane:20527', u'10.11.5.80:kss-automap-lane:20528', u'10.11.5.43:kss-automap-lane:20530', u'10.11.5.43:kss-automap-lane:20531', u'10.11.5.78:kss-automap-lane:20531', u'10.11.5.78:kss-automap-lane:20530', u'10.11.5.39:kss-automap-lane:20529', u'10.11.5.39:kss-automap-lane:20528', u'10.11.5.39:kss-automap-lane:20527', u'10.11.5.17:kss-automap-lane:20529', u'10.11.5.17:kss-automap-lane:20528', u'10.11.5.17:kss-automap-lane:20527', u'10.11.5.43:kss-automap-lane:20527', u'10.11.5.43:kss-automap-lane:20528', u'10.11.5.43:kss-automap-lane:20529', u'10.11.5.51:kss-automap-lane:20528', u'10.11.5.51:kss-automap-lane:20527', u'10.11.5.34:kss-automap-lane:20529', u'10.11.5.34:kss-automap-lane:20528', u'10.11.5.34:kss-automap-lane:20527', u'10.11.5.51:kss-automap-lane:20529', u'10.11.5.34:kss-automap-lane:20531', u'10.11.5.34:kss-automap-lane:20530', u'10.11.5.75:kss-automap-lane:20529', u'10.11.5.75:kss-automap-lane:20527', u'10.11.5.75:kss-automap-lane:20528', u'10.11.5.62:kss-automap-lane:20529', u'10.11.5.36:kss-automap-lane:20527', u'10.11.5.75:kss-automap-lane:20530', u'10.11.5.75:kss-automap-lane:20531', u'10.11.5.62:kss-automap-lane:20528', u'10.11.5.62:kss-automap-lane:20527', u'10.11.5.84:kss-automap-lane:20531', u'10.11.5.84:kss-automap-lane:20530', u'10.11.5.86:kss-automap-lane:20527', u'10.11.5.86:kss-automap-lane:20528', u'10.11.5.86:kss-automap-lane:20529', u'10.11.5.19:kss-automap-lane:20527', u'10.11.5.19:kss-automap-lane:20528', u'10.11.5.36:kss-automap-lane:20528', u'10.11.5.19:kss-automap-lane:20529', u'10.11.5.36:kss-automap-lane:20529', u'10.11.5.84:kss-automap-lane:20529', u'10.11.5.36:kss-automap-lane:20531', u'10.11.5.4:kss-automap-lane:20529', u'10.11.5.51:kss-automap-lane:20531', u'10.11.5.36:kss-automap-lane:20530', u'10.11.5.51:kss-automap-lane:20530', u'10.11.5.84:kss-automap-lane:20528', u'10.11.5.84:kss-automap-lane:20527', u'10.11.5.86:kss-automap-lane:20530', u'10.11.5.86:kss-automap-lane:20531', u'10.11.5.73:kss-automap-lane:20529', u'10.11.5.12:kss-automap-lane:20529', u'10.11.5.12:kss-automap-lane:20528', u'10.11.5.62:kss-automap-lane:20531', u'10.11.5.12:kss-automap-lane:20527', u'10.11.5.62:kss-automap-lane:20530', u'10.11.5.4:kss-automap-lane:20528', u'10.11.5.4:kss-automap-lane:20527', u'10.11.5.73:kss-automap-lane:20528', u'10.11.5.73:kss-automap-lane:20527']
[Finish] Success to check [kss-automap-lane]